app/src/main/java/mozilla/lockbox/LockboxApplication.kt

```diff
@@ -12,13 +12,17 @@ import io.sentry.android.AndroidSentryClientFactory
 import mozilla.components.support.base.log.Log
 import mozilla.components.support.base.log.logger.Logger
 import mozilla.components.support.base.log.sink.AndroidLogSink
+import mozilla.lockbox.store.TelemetryStore
 
 val log: Logger = Logger("Lockbox")
 class LockboxApplication : Application() {
     override fun onCreate() {
         super.onCreate()
         Log.addSink(AndroidLogSink())
 
+        // hook this context into Telemetry
+        TelemetryStore.shared.applyContext(this)
```

Comment:
I think there's a larger question about how to get context in stores where it's required... @mihainisipeanusv is dealing with the same issue in #124 right now. It might be worth having a discussion about this rather than addressing in these two PRs separately.

Reply:
@sashei (et al @mozilla-lockbox/mobile-engineering ).

That's reasonable, although it looks like we're following the same pattern with slightly different names.

Unless there's objections to the overall approach, I suggest the discussion start with:

1. agree to the pattern @mihainisipeanusv and I seem to have come to already, with the same names
2. encapsulate it into a common base class for these types of Stores (e.g. `ContextualStore` ?)